tests/asset_loading.rs

```diff
@@ -0,0 +1,114 @@
+//! This module tests basic asset loading.
+
+extern crate core;
+
+use bevy::prelude::*;
+use bevy_internal::asset::{Asset, LoadState};
+use bevy_internal::gltf::GltfPlugin;
+use bevy_internal::log::LogPlugin;
+use bevy_internal::render::mesh::MeshPlugin;
+use bevy_internal::render::render_resource::ShaderLoader;
+use bevy_internal::scene::SceneLoader;
+
+#[cfg(feature = "png")]
+#[test]
+fn load_png() {
+    let mut app = setup_test_app();
+
+    let image: &Image = assert_asset_loads(&mut app, "load_tests/bevy_pixel.png");
+    assert_eq!(image.size(), Vec2::new(128., 96.));
```

Comment:
I think we should refactor these verifcation steps out: it's very plausible that we'll want to add other checks (e.g. for colors) later.